packages/common/src/ui-common/environment/environment.ts

```diff
@@ -2,6 +2,7 @@ import { Clock } from "../datetime/clock";
 import { HttpClient } from "../http";
 import { Localizer } from "../localization";
 import type { Logger, LoggerFactory, LoggingContext } from "../logging";
+import { Notification as Notifier } from "../notification";
```

Comment:
This is probably a pain, but maybe it's worth renaming "Notifcation" to "Notifier" systematically to distinguish a service that provides notifications from the notifications themselves and avoid confusion in the future.

Reply:
Yeah good point, I'll update that too.